Source/WebGPU/WebGPU/BindGroupLayout.mm

```diff
@@ -141,13 +140,34 @@
     if (storageTexture.nextInChain)
         return nil;
 
-    UNUSED_PARAM(storageTexture);
     auto descriptor = [MTLArgumentDescriptor new];
     descriptor.dataType = MTLDataTypeTexture;
     // FIXME: Implement this.
     return descriptor;
 }
 
+bool BindGroupLayout::isPresent(const WGPUExternalTextureBindingLayout&)
+{
+    return true;
+}
+
+static MTLArgumentDescriptor *createArgumentDescriptor(const WGPUExternalTextureBindingLayout& externalTexture)
+{
+    if (externalTexture.nextInChain)
+        return nil;
+
+    auto descriptor = [MTLArgumentDescriptor new];
+    descriptor.dataType = MTLDataTypeTexture;
+#if USE(METAL_ARGUMENT_ACCESS_ENUMS)
+ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+    descriptor.access = MTLArgumentAccessReadOnly;
+ALLOW_DEPRECATED_DECLARATIONS_END
+#else
+    descriptor.access = MTLBindingAccessReadOnly;
+#endif
+    return descriptor;
```

Comment:
This is wrong - we'll need to add tons of stuff to the argument encoder here, not just a single texture. I think that should happen in another patch, though, so I think the only thing to change immediately here is to add a FIXME.